src/utils/api/helper.ts

```diff
@@ -436,7 +334,7 @@ export const activePaymentTask = (
     )
   );
 
-export const getTransactionData = async ({
+const getTransactionData = async ({
```

Comment:
why remove export?

Reply:
This const is not used outside, so I thought  it was unnecessary